src/lib.rs

```diff
@@ -86,8 +86,8 @@
 //!   [ring](https://github.com/briansmith/ring) crate.
 //! * `serde`: Enables serde serialization for a number of basic types.
 //! * `sign`: basic DNSSEC signing support. This will enable the
-#![cfg_attr(feature = "sign", doc = "  [sign]")]
-#![cfg_attr(not(feature = "sign"), doc = "  sign")]
+#![cfg_attr(feature = "unstable-sign", doc = "  [sign]")]
```

Comment:
Update: I have moved this GH comment down to the correct part of the file.

The `sign` on the right should also be changed to `unstable-sign`. Same issue below for other renamed features. Also, now unstable features that previoulsy were not marked as such should be moved down the doc to the unstable features section.

Reply:
The `sign` on the right refers to the module, whose name has not changed.  I'll move these down to the unstable feature list.